src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -35,12 +38,31 @@ attributes:
  persistVolumes: ‘false’
 ----
 
-.Asynchronous storage
+=== Comparison table between I/O of  Ephemeral (`emptyDir`) vs  Persitent (`EBS` more about link:https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs[AWS EBS])
+
+[cols="3", options="header"]
+|===
+|Command
+|Ephemeral
+|Persitent
+
+|Clone  Eclipse Che
+|0m 19s
+|1m 26s
+
+|Generate 1000 random files
+|1m 12s
+|44m 53s
+|===
 
-This is a combination of the above two. The initial workspace container will mount an emptyDir, however  your changes will be restored on workspace start and a backup is performed on workspace stop. This storage type also offers fast I/O (similar to an ephemeral mode) however, workspace project changes are persisted. Keep in mind that this is still an experimental feature.
+.Asynchronous storage (Experimental)
 
-Synchronization is powered by ‘rsync’ via SSH protocol. When a workspace is configured with this storage type, a special plugin is automatically added to the workspace configuration. This plugin will run ‘rsync’ command on workspace start to restore changes if they exist. When a workspace is stopped, it will send changes to the permanent storage. Rsync process can take some time, so a workspace will not start immediately like in case with a persistent and ephemeral storage. For relatively small projects the restore procedure is completed very fast and you will see project source files and folders immediately after Che Theia is initialized. If rsync takes longer time, the synchronization process will be shown in Che Theia UI status bar area.
+This is a combination of the above two. The initial workspace container will mount an emptyDir, however, the changes will be restored on workspace start and a backup is performed on workspace stop. This storage type also offers fast I/O (similar to the ephemeral mode) and workspace project changes are persisted.
 
+Synchronization is powered by ‘rsync’ (link:https://rsync.samba.org/[]) via SSH (link:https://www.openssh.com/[]) protocol. When a workspace is configured with this storage type, a special plugin is automatically added to the workspace configuration (link:https://github.com/che-incubator/workspace-data-sync/][source on GitHub]). This plugin will run ‘rsync’ command on workspace start to restore changes if they exist. When a workspace is stopped, it will send changes to the permanent storage. Rsync process can take some time, so a workspace will not start immediately like in case with a persistent and ephemeral storage. For relatively small projects the restore procedure is completed very fast and you will see project source files and folders immediately after Che Theia is initialized. If rsync takes longer time, the synchronization process will be shown in Che Theia UI status bar area. (link:https://github.com/eclipse/che-theia/tree/master/extensions/eclipse-che-theia-file-sync-tracker][Extension in Che Theia repository]).
```

Comment:
```suggestion
Synchronization is powered by ‘rsync’ (link:https://rsync.samba.org/[]) via SSH (link:https://www.openssh.com/[]) protocol. When a workspace is configured with this storage type, the link:https://github.com/che-incubator/workspace-data-sync/[workspace-data-sync] plugin is automatically added to the workspace configuration. This plugin will run ‘rsync’ command on workspace start to restore changes if they exist. When a workspace is stopped, it will send changes to the permanent storage. The rsync process can take some time -  for relatively small projects the restore procedure is completed very fast and you will see project source files and folders immediately after Che Theia is initialized. If rsync takes a longer time, the synchronization process will be shown in Che Theia UI status bar area. (link:https://github.com/eclipse/che-theia/tree/master/extensions/eclipse-che-theia-file-sync-tracker][Extension in Che Theia repository]).
```